app-modules/interaction/resources/views/components/interaction-timeline-item.blade.php

```diff
@@ -0,0 +1,82 @@
+{{--
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+--}}
+@use('App\Filament\Resources\UserResource')
+<div>
+    <div class="flex flex-row justify-between">
+        <h3 class="mb-1 flex items-center text-lg font-semibold text-gray-500 dark:text-gray-100">
+            <a
+                class="font-medium underline"
+                href="{{ UserResource::getUrl('view', ['record' => $record->user]) }}"
+            >
+                {{ $record->user->name }}
+            </a>
+            <span class="ml-2 flex space-x-2">
+                <div class="relative">
+                    <x-filament::icon
+                        class="h-5 w-5 text-gray-400 dark:text-gray-100"
+                        icon="heroicon-o-pencil-square"
+                    />
+                </div>
+            </span>
+        </h3>
+
+        <div>
+            {{ $viewRecordIcon }}
+        </div>
+    </div>
+
+    <time class="mb-2 block text-sm font-normal leading-none text-gray-400 dark:text-gray-500">
+        {{ $record?->type?->name }}
+    </time>
+
+    <time class="mb-2 block text-sm font-normal leading-none text-gray-400 dark:text-gray-500">
+        Sent {{ $record->created_at->diffForHumans() }}
```

Comment:
I think this is actually supposed to be `start_datetime` and `end_datetime`. Also "Sent" doesn't make sense here as it's meant to display the date/time that this interaction happened.

Reply:
> I think this is actually supposed to be `start_datetime` and `end_datetime`. Also "Sent" doesn't make sense here as it's meant to display the date/time that this interaction happened.

Hmm ! as per screenshot it's displaying this

so, on text as per screenshot `1 week ago` we need to display start_datetime? and where to display the `end_datetime` on pop_up? when click on view icon?

![image](https://github.com/user-attachments/assets/c4f7d7b1-1570-4cea-b54e-2dc9783175d7)

Reply:
Ah got it. I don't think we need to do the end_datetime. But rather than making this time based on `created_at` kets do `start_datetime` which is the actual recorded time of the interaction.

If you want you can confirm with Product what is wanted here